stoqs/stoqs/templates/stoqs/stoqsquery.html

```diff
@@ -3274,7 +3274,7 @@
         var vrHMD = null, vrSensor = null;
 
 
-        if ( rifted ) {
+        if ( rifted || cardboarded ) {
```

Comment:
I think I steered you wrong with this suggestion. It doesn't actually do anything as it's triggered only by the `rift-button` class. It's probably best to change it back to the way it was.